recipes-bsp/firmware-qca/firmware-qca.inc

```diff
@@ -1,15 +1,15 @@
-# Copyright 2018 NXP
+# Copyright 2018-2019 NXP
 
 SUMMARY = "Qualcomm Wi-Fi and Bluetooth firmware"
 SECTION = "base"
 LICENSE = "Proprietary"
-LIC_FILES_CHKSUM = "file://COPYING;md5=6dfb32a488e5fd6bae52fbf6c7ebb086 \
+LIC_FILES_CHKSUM = "file://COPYING;md5=6c12031a11b81db21cdfe0be88cac4b3 \
```

Comment:
@panfir What is the reason MD5 is updated here? Are license terms stayed the same and it is just a License header update?

It is not obvious since it is not indicated in the commit message why this is needed.

Reply:
> @panfir What is the reason MD5 is updated here? Are license terms stayed the same and it is just a License header update?
> 
> It is not obvious since it is not indicated in the commit message why this is needed.

The EULA file has been changed to August 2019 version.